includes/class-wc-gateway-ppec-address.php

```diff
@@ -581,61 +835,169 @@ public function normalizeState() {
+				'indiana'                                 => 'IN',
+				'iowa'                                    => 'IA',
+				'kansas'                                  => 'KS',
+				'kentucky'                                => 'KY',
+				'louisiana'                               => 'LA',
+				'maine'                                   => 'ME',
+				'maryland'                                => 'MD',
+				'massachusetts'                           => 'MA',
+				'michigan'                                => 'MI',
+				'minnesota'                               => 'MN',
+				'mississippi'                             => 'MS',
+				'missouri'                                => 'MO',
+				'montana'                                 => 'MT',
+				'nebraska'                                => 'NE',
+				'nevada'                                  => 'NV',
+				'new hampshire'                           => 'NH',
+				'new jersey'                              => 'NJ',
+				'new mexico'                              => 'NM',
+				'new jersey'                              => 'NJ',
+				'new mexico'                              => 'NM',
+				'new york'                                => 'NY',
+				'north carolina'                          => 'NC',
+				'north dakota'                            => 'ND',
+				'ohio'                                    => 'OH',
+				'oklahoma'                                => 'OK',
+				'oregon'                                  => 'OR',
+				'pennsylvania'                            => 'PA',
+				'puerto rico'                             => 'PR',
+				'rhode island'                            => 'RI',
+				'south carolina'                          => 'SC',
+				'south dakota'                            => 'SD',
+				'tennessee'                               => 'TN',
+				'texas'                                   => 'TX',
+				'utah'                                    => 'UT',
+				'vermont'                                 => 'VT',
+				'virginia'                                => 'VA',
+				'washington'                              => 'WA',
+				'west virginia'                           => 'WV',
+				'wisconsin'                               => 'WI',
+				'wyoming'                                 => 'WY',
+				'armed forces americas'                   => 'AA',
+				'armed forces'                            => 'AE',
+				'armed forces pacific'                    => 'AP',
+				'american samoa'                          => 'AS',
+				'guam'                                    => 'GU',
+				'northern mariana islands'                => 'MP',
+				'virgin islands'                          => 'VI',
+				// The next few derivations don't show up on the list, but seem obvious
+				'us virgin islands'                       => 'VI',
+				'u.s. virgin islands'                     => 'VI',
+				'u s virgin islands'                      => 'VI',
+				'u. s. virgin islands'                    => 'VI'
 			);
 		}
 
+		error_log( strtolower(trim($this->_state)));
```

Comment:
@menakas there's an error log that slipped in here.

Reply:
Oops. Thanks James. I removed it.